### ISSUE TITLE
What do we do about the MDA logo license?

### ISSUE THREAD

**Issue by IAlibay** [MDAnalysis/cookiecutter-mdakit#87]
Place it in _static for now?

**Comment by orbeckst** [comment]
Yes, just put it there and make clear that it relates to one specific file.

**Comment by orbeckst** [comment]
We're using the placeholder kits logo https://github.com/MDAnalysis/cookiecutter-mdakit/tree/main/%7B%7Bcookiecutter.repo_name%7D%7D/docs/source/_static/logo so this issue is not relevant anymore.
